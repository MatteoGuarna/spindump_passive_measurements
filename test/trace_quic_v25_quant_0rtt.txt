Testing with and without 0-RTT on Quant. 
