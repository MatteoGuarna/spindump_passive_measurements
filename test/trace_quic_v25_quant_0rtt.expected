QUIC 100.72.46.172 <-> 13.48.145.10 afe4e92bb8407a26046796-null (52304:4433) at 1580745149376299 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 afe4e92bb8407a26046796-6796493d08afe4 (52304:4433) at 1580745149410878 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 afe4e92bb8407a26046796-6796493d08afe4 (52304:4433) at 1580745149410878 measurement starting right 34579 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-6796493d08afe4 (52304:4433) at 1580745149412345 measurement starting left 1467 packets 1 1 bytes 1228 55 bandwidth 1228 55 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-04a0845c (52304:4433) at 1580745149449353 change starting packets 2 1 bytes 2456 55 bandwidth 2456 55 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 null-04a0845c (52304:4433) at 1580745149449353 measurement up right 37008 packets 2 1 bytes 2456 55 bandwidth 2456 55 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 e3344b5d-04a0845c (52304:4433) at 1580745149449359 packet up responder length 1280 packets 2 3 bytes 2456 2518 bandwidth 2456 2518 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 e3344b5d-04a0845c (52304:4433) at 1580745149449968 packet up responder length 1280 packets 2 4 bytes 2456 3798 bandwidth 2456 3798 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 e3344b5d-04a0845c (52304:4433) at 1580745149451891 packet up initiator length 68 packets 3 4 bytes 2524 3798 bandwidth 2524 3798 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 e3344b5d-04a0845c (52304:4433) at 1580745149452168 packet up initiator length 69 packets 4 4 bytes 2593 3798 bandwidth 2593 3798 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 e3344b5d-04a0845c (52304:4433) at 1580745149453329 packet up initiator length 1500 packets 5 4 bytes 4093 3798 bandwidth 4093 3798 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 52dcc51b-04a0845c (52304:4433) at 1580745149488910 packet up responder length 70 packets 5 5 bytes 4093 3868 bandwidth 4093 3868 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 52dcc51b-04a0845c (52304:4433) at 1580745149489222 packet up responder length 1500 packets 5 6 bytes 4093 5368 bandwidth 4093 5368 note "V.qn25,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04f39d7e-a22121708ffa1d2060d6f3059231761f (61694:4433) at 1580745155249331 new starting packets 0 0 bytes 0 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04f39d7e-04168220 (61694:4433) at 1580745155288501 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04f39d7e-04168220 (61694:4433) at 1580745155288501 measurement up right 39170 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 f39d7e96-04168220 (61694:4433) at 1580745155292136 packet up initiator length 144 packets 2 1 bytes 1372 1500 bandwidth 1372 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 f39d7e96-04168220 (61694:4433) at 1580745155292540 packet up initiator length 1500 packets 3 1 bytes 2872 1500 bandwidth 2872 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 f39d7e96-04168220 (61694:4433) at 1580745155327335 packet up responder length 70 packets 3 2 bytes 2872 1570 bandwidth 2872 1570 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 7c70cee0-04168220 (61694:4433) at 1580745155328193 packet up responder length 70 packets 3 5 bytes 2872 3863 bandwidth 2872 3863 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 043b2577-b2cc27f54964198512f2a0304fdeddfa0030b0d5 (58991:4433) at 1580745156231837 new starting packets 0 0 bytes 0 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 043b2577-04e47803 (58991:4433) at 1580745156270708 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 043b2577-04e47803 (58991:4433) at 1580745156270708 measurement up right 38871 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 3b257758-04e47803 (58991:4433) at 1580745156273864 packet up initiator length 144 packets 2 1 bytes 1372 1500 bandwidth 1372 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 3b257758-04e47803 (58991:4433) at 1580745156274255 packet up initiator length 1500 packets 3 1 bytes 2872 1500 bandwidth 2872 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 3b257758-04e47803 (58991:4433) at 1580745156308901 packet up responder length 70 packets 3 2 bytes 2872 1570 bandwidth 2872 1570 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 6c54ec05-04e47803 (58991:4433) at 1580745156311352 packet up responder length 70 packets 4 5 bytes 2932 3863 bandwidth 2932 3863 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04dcb59a-2d80dac027a2763712 (59941:4433) at 1580745157689620 new starting packets 0 0 bytes 0 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04dcb59a-04c9221a (59941:4433) at 1580745157729141 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 04dcb59a-04c9221a (59941:4433) at 1580745157729141 measurement up right 39521 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 dcb59a89-04c9221a (59941:4433) at 1580745157731860 packet up initiator length 144 packets 2 1 bytes 1372 1500 bandwidth 1372 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 dcb59a89-04c9221a (59941:4433) at 1580745157732311 packet up initiator length 1500 packets 3 1 bytes 2872 1500 bandwidth 2872 1500 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 dcb59a89-04c9221a (59941:4433) at 1580745157765770 packet up responder length 70 packets 3 2 bytes 2872 1570 bandwidth 2872 1570 note "V.qn25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 13.48.145.10 4083e3e6-04c9221a (59941:4433) at 1580745157766378 packet up responder length 70 packets 3 5 bytes 2872 3863 bandwidth 2872 3863 note "V.qn25,0-RTT,no spin"
